plugins/in_calyptia_fleet/in_calyptia_fleet.c

```diff
@@ -1327,210 +1297,187 @@ static struct cfl_array *read_glob(const char *path)
 
 #endif
 
-static int cfl_array_qsort_conf_files(const void *arg_a, const void *arg_b)
-{
-    struct cfl_variant *var_a = (struct cfl_variant *)*(void **)arg_a;
-    struct cfl_variant *var_b = (struct cfl_variant *)*(void **)arg_b;
-
-    if (var_a == NULL && var_b == NULL) {
-        return 0;
-    }
-    else if (var_a == NULL) {
-        return -1;
-    }
-    else if (var_b == NULL) {
-        return 1;
-    }
-    else if (var_a->type != CFL_VARIANT_STRING &&
-             var_b->type != CFL_VARIANT_STRING) {
-        return 0;
-    }
-    else if (var_a->type != CFL_VARIANT_STRING) {
-        return -1;
-    }
-    else if (var_b->type != CFL_VARIANT_STRING) {
-        return 1;
-    }
-
-    return strcmp(var_a->data.as_string, var_b->data.as_string);
-}
-
-static int calyptia_config_delete_old_dir(const char *cfgpath)
```

Comment:
Replaced by `calyptia_config_delete_by_symlink`, which no longer relies on implicit ordering of files in the config directory.